kotlin-sqlobject/src/main/kotlin/org/jdbi/v3/sqlobject/kotlin/KotlinDefaultMethodHandlerFactory.kt

```diff
@@ -32,13 +33,15 @@ class KotlinDefaultMethodHandlerFactory : ExtensionHandlerFactory {
         val implementation = getImplementation(sqlObjectType, method) ?: return Optional.empty()
 
         return Optional.of(
-            ExtensionHandler { _, t, a ->
-                @Suppress("SwallowedException")
-                try {
-                    @Suppress("SpreadOperator")
-                    implementation.invoke(null, t, *a)
-                } catch (e: InvocationTargetException) {
-                    throw e.targetException
+            ExtensionHandler { c, t ->
```

Comment:
Kotlin still hates you: 

`Use "_" instead of this unused lambda parameter "c".`

:-)